website/settings/defaults.py

```diff
@@ -283,3 +284,4 @@ def parent_dir(path):
 
 ENABLE_ARCHIVER = True
 ###########################
+REGISTRATION_APPROVAL_PERIOD = datetime.timedelta(days=2)
```

Comment:
Why do we need both `REGISTRATION_APPROVAL_TIME` and `REGISTRATION_APPROVAL_PERIOD`?